Cargo.toml

```diff
@@ -270,3 +270,7 @@ arrow-row = { git = "https://github.com/apache/arrow-rs.git", branch = "master"
 arrow-schema = { git = "https://github.com/apache/arrow-rs.git", branch = "master" }
 arrow-select = { git = "https://github.com/apache/arrow-rs.git", branch = "master" }
 arrow-string = { git = "https://github.com/apache/arrow-rs.git", branch = "master" }
+
+[patch."https://github.com/datafusion-contrib/datafusion-federation.git"]
+# Can remove once https://github.com/datafusion-contrib/datafusion-federation/pull/81 merges
+datafusion-flight-sql-server = { path = "../datafusion-federation/datafusion-flight-sql-server" }
```

Comment:
Looks like this should be your repo for now: `datafusion-flight-sql-server = { git = "https://github.com/nathanielc/datafusion-federation.git", branch = "feat/put-prepared" }`

Reply:
Yeah, I should point at my branch... However reviews on that repo are pretty fast so I am hoping that it gets reviewed before this is ready to merge so that I can just point to main instead of my branch. If not I'll update here.